modules/platforms/dotnet/Apache.Ignite.Core/Impl/Client/ClientFailoverSocket.cs

```diff
@@ -553,6 +553,12 @@ private object ReadDistributionMapResponse(IBinaryStream s)
 
                 // Count partitions to avoid reallocating array.
                 int maxPartNum = 0;
+
+                if (grp.PartitionMap == null)
+                {
+                    continue;
+                }
```

Comment:
Not sure the code is following code style here. Also, will this change break backward compatibility for .NET?

Reply:
I'm not sure if our java code style applies here, so I just looked up other null check examples and copied it
![image](https://user-images.githubusercontent.com/4058545/88535655-febcdb80-d012-11ea-8728-9015edc82f20.png)